src/containers/AppRoutes/AppRoutes.tsx

```diff
@@ -12,12 +13,33 @@ import LegacySeries from '#src/pages/LegacySeries/LegacySeries';
 import MediaScreenRouter from '#src/pages/ScreenRouting/MediaScreenRouter';
 import PlaylistScreenRouter from '#src/pages/ScreenRouting/PlaylistScreenRouter';
 import Layout from '#src/containers/Layout/Layout';
+import Profiles from '#src/containers/Profiles/Profiles';
+import CreateProfile from '#src/containers/Profiles/CreateProfile';
+import { useConfigStore } from '#src/stores/ConfigStore';
+import { useAccountStore } from '#src/stores/AccountStore';
+import EditProfile from '#src/containers/Profiles/EditProfile';
+import useQueryParam from '#src/hooks/useQueryParam';
+import { useProfilesFeatureEnabled } from '#src/hooks/useProfiles';
+import { useProfileStore } from '#src/stores/ProfileStore';
 
 export default function AppRoutes() {
   const { t } = useTranslation('error');
+  const userModal = useQueryParam('u');
+
+  const { accessModel } = useConfigStore(({ config, accessModel }) => ({ config, accessModel }), shallow);
+  const user = useAccountStore(({ user }) => user, shallow);
+  const { profile } = useProfileStore();
+  const profilesEnabled = useProfilesFeatureEnabled();
+
+  const shouldManageProfiles = !!user && profilesEnabled && !profile && (accessModel === 'SVOD' || accessModel === 'AUTHVOD') && !userModal;
 
   return (
     <Routes>
+      <Route index={shouldManageProfiles} element={<Navigate to="/u/profiles" />} />
```

Comment:
This only works when the user navigates to the root `/`. Should this also be working for deep links?

A solution to this would be to directly redirect and pass a `?goto` query param so that the user ends up on the expected page after choosing a profile.